packages/react-core/src/evm/providers/thirdweb-sdk-provider.tsx

```diff
@@ -218,7 +217,7 @@ export const ThirdwebSDKProvider = <
   // @ts-expect-error - different subtype of Chain[] but this works fine
   supportedChains = defaultChains,
   activeChain,
-  thirdwebApiKey = DEFAULT_API_KEY,
```

Comment:
react-core also has to have (its own?) api key since it can be used independently